model/lifecycle_test.go

```diff
@@ -2587,10 +2607,13 @@ func TestAddNewTasks(t *testing.T) {
 			assert.NoError(t, err)
 			activatedTasks, err := task.FindAll(db.Query(bson.M{task.ActivatedKey: true}))
 			assert.NoError(t, err)
+			build, err := build.FindOneId("b0")
+			assert.NoError(t, err)
```

Comment:
Is this needed? Seems like we don't need to double-check for the existence of `b0`.

Reply:
We need to pull it from the db to check if addNewTasks activated the build.

Reply:
Ah I see. Does it need to also check if `build` is non-nil? Just checking the error won't check for the doc not found case.

Reply:
The test will panic on that line if `build` is nil, but I kind of prefer that tests don't panic if they don't have to (because they stop the entire test runtime, rather than just this test). I don't think this is a hard requirement given how often the existing tests _can_ panic, but it is part of the style guide and it would be nice if we could aim for less panicking in tests.

Reply:
good point, fixed